taskcluster/scriptworker-task.yml

```diff
@@ -6,18 +6,22 @@ build:
     - "linux-amd64-cpu-opt"
     - "linux-amd64-gpu-opt"
     - "linux-rpi3-cpu-opt"
+    - "node-package"
   allowed:
     - "push"
   ref_match: "refs/tags/"
   routes:
     - "notify.irc-channel.${notifications.irc}.on-exception"
     - "notify.irc-channel.${notifications.irc}.on-failed"
   artifacts_deps:
-    - "darwin-amd64-cpu-opt"
-    - "linux-amd64-cpu-opt"
-    - "linux-amd64-gpu-opt"
-    - "linux-rpi3-cpu-opt"
-    - "node-package"
+    python:
+      - "darwin-amd64-cpu-opt"
+      - "linux-amd64-cpu-opt"
+      - "linux-amd64-gpu-opt"
+      - "linux-rpi3-cpu-opt"
+    javascript:
+      - "linux-amd64-gpu-opt"
```

Comment:
Shouldn't we have JS set being a superset of Python one?

Reply:
Never mind. "node-package" takes care of putting all together darwin-amd64-cpu, linux-amd64-cpu and linux-rpi3.

Might be worth a comment then, if I got tricked :)